xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/dto/response/KucoinKline.java

```diff
@@ -0,0 +1,77 @@
+package org.knowm.xchange.kucoin.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.kucoin.dto.KlineIntervalType;
+
+import java.math.BigDecimal;
+
+@Data
```

Comment:
this adds getters and setters which you then also added manually below

DTOs should be value objects so either change this annotation to `@Value` and remove those implicit getters bellow

or dont use lombok and make all fields final with just getters

Reply:
Ok, makes sense.